mw-webapp/src/component/footer/Footer.module.scss

```diff
@@ -1,52 +1,72 @@
 @import "src/styles/variables";
 
-$userNameFontSize: 16px;
+$logoHeight: 55px;
+$iconSize: 32px;
+$privacyLineHeight: 140%;
+$socialTextFontWeight: 500;
+$rulerHeight: 1px;
 
 .footer {
   background-color: var(--quaternaryBgColor);
 }
 
 .footerWrapper {
-  width: 100%;
   max-width: $maxWidthContainer;
-  justify-content: space-between;
-  padding: $paddingBig $paddingMedium;
-  border-bottom: $primaryBorder solid var(--secondaryStrokeColor);
+  box-sizing: content-box;
+  flex-direction: column;
+  padding: 0 $paddingMedium;
   margin: 0 auto;
+  gap: 0;
+}
+
+.socialLinks {
+  align-items: flex-start;
+  justify-content: space-between;
+  padding: $paddingBig 0;
+}
+
+.ruler {
+  height: $rulerHeight;
+  border: none;
+  background-color: var(--secondaryStrokeColor);
```

Comment:
if it's background then it's not Stroke

Reply:
Is it possible to use border-top or border-bottom instead of background color?

Reply:
I did as you said, but I like previous version better. Because when you read a code, you see that there is a ruler, it has 1px of height, it has a color, no borders. And semantically ruler is a stroke, I don't see any problems to use background-color for it